apiserver/facades/client/usermanager/usermanager.go

```diff
@@ -90,24 +93,23 @@ func NewAPI(
 	apiUser coreuser.User,
 	isAdmin bool,
 	logger corelogger.Logger,
-	controllerUUID string,
+	controllerTag names.ControllerTag,
```

Comment:
It would theoretically make more sense to keep taking the UUID and not the tag as our plan is to just have tag as an over the wire construct so unless we are writing a response or reading one our use cases for having to use a tag internally will vastly diminish the more we move forward.

For example now we have a use case of using a tag to go back to a uuid and before we had a uuid going into a tag. The cases of tag to uuid are only going to increase while the other will decrease so start passing the uuid now as that is what we will continue to naturally work with. Especially once the authz gets some love.


Just my two cents.

Reply:
Seems we already have more cases of needing the uuid than we do the tag. The uuid was plumbed through as we moved to Dqlite to support this effort. Seems weird to undo that now as well.